packages/@azure/servicebus/data-plane/test/renewLockSessions.spec.ts

```diff
@@ -438,8 +437,8 @@ async function testAutoLockRenewalConfigBehavior(
         await delay(options.delayBeforeAttemptingToCompleteMessageInSeconds * 1000);
 
         let errorWasThrown: boolean = false;
-        await brokeredMessage.complete().catch((err) => {
-          errorWasThrown = true; // Service bus completes the message even when the session lock expires.
```

Comment:
Can we have this comment back, just for completion sake?

Reply:
Got it, updated